calico_kubernetes/calico_kubernetes.py

```diff
@@ -35,6 +35,9 @@
 KUBE_API_ROOT = os.environ.get('KUBE_API_ROOT',
                                'http://kubernetes-master:8080/api/v1/')
 
+# Allow the user to enable/disable namespace isolation policy
+NAMESPACE_ISOLATION = os.environ.get('NAMESPACE_ISOLATION', 'true')
```

Comment:
I think calling this NAMESPACE_ISOLATION is a bit confusing, since namespace isolation will still be in effect if policy is declared, just not when no policy is declared.

A more apt name might be something like "DEFAULT_POLICY", with options for "ns_isolation", "allow".  This means we could potentially support multiple other default behaviors in the future.